### DISCUSSION TITLE
Add a choice of your own font for quotes to the theme settings

### DISCUSSION THREAD

**Discussion by irmod** [damionx7/Zettel-Notes-Documentation#166] [Feature Requests]
In my notes, I have long used the "Georgia" font to display quotes.

"Zettel Notes" has convenient theme settings.

Is it possible to add your own font for Blockquote in Theme --> Markdown Style --> Edit Theme.

![Blockquote_Font.jpg](https://github.com/user-attachments/assets/18ae2c61-a62a-43d6-aae8-9dcea7948e63)

**Comment by damionx7** [comment]
Thanks for the feature request. I will add it.

**Comment by irmod** [comment]
@damionx7 

I'm really, really looking forward to adding my own quote font to the
theme... Please!

**Comment by irmod** [comment]
In version 2.7.6 @damionx7 added a custom font setting for Blockquote.  Enjoy!!!
